app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/SavingThrow.java

```diff
@@ -0,0 +1,43 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+class SavingThrow {
```

Comment:
Proposal for future:
Function that takes a list argument and calculates whether to give advantage, disadvantage or nothing.